data_pipeline/README.md

```diff
@@ -30,14 +29,69 @@ We will publish the code to generate the speed classifications from the traffic
 | London  | Highways England |[Highways England network journey time and traffic flow data](https://data.gov.uk/dataset/9562c512-4a0b-45ee-b6ad-afc0f99b841f/highways-england-network-journey-time-and-traffic-flow-data) | [Traffic Flow data - Sites and Historical Reports](https://webtris.highwaysengland.co.uk/)|  [Open Government Licence](https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/) |
 
 
+### Vehicle Counter Data Pipeline
+
+Vehicle loop counter data can be processed using the pipeline used for counter validations in MeTS-10: https://github.com/iarai/MeTS-10/tree/master/analysis/val02_counters
+
+This process requires the road graph data in `road_graph/<city>`.
+
+* `counters01_prepare_data.ipynb`: Downloads and processes the counter readings. Outputs `loop_counters/<CITY>/counters_normalized_{month}.parquet`
+* `counters02_match_counters.ipynb`: Merges the loop counter locations with the road graph and outputs `loop_counters/<CITY>/counters_matched.parquet`
```

Comment:
Should it not output a new road graph with the new nodes as well? (see CODE-1 in overleaf)

Reply:
added TODO